src/debugAdapter/goDebug.ts

```diff
@@ -321,21 +321,36 @@ class GoDebugSession extends DebugSession {
 		log('InitializeEvent');
 	}
 
+	protected findPathSeperator(path) {
+		if (/^(\w:[\\/]|\\\\)/.test(path)) return '\\';
```

Comment:
Not sure how that would help. Trying to match absolute Windows paths like C:\ or C:/ and share paths like \\server\share. It also matches \\?\, so it does not need it's own check.